File: exercises/beecrowd/1016/1016.ts
// https://judge.beecrowd.com/pt/problems/view/1016
import * as fs from 'fs';

const input = fs.readFileSync('/dev/stdin', 'utf8');
const lines = input.split(/\r?\n/);

const km = Number(lines.shift());

console.log(`${km*2} minutos`);

File: exercises/beecrowd/1016/1016.java
// https://judge.beecrowd.com/pt/problems/view/1016
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int x = sc.nextInt();
        int minutos = 2 * x;
        System.out.println(minutos + " minutos");
        sc.close();
    }
}
